runtime/doc/news.txt

```diff
@@ -33,7 +33,7 @@ DEFAULTS
 
 EDITOR
 
-• TODO
+• Signs are now drawn from left to right
```

Comment:
need to say a bit about the motivation, otherwise it just sounds random
```suggestion
• Signs are drawn from left to right, so the "highest priority" appear leftmost, while any signs exceeding the signcolumn width (thus not displayed) are the also the lowest priority.
```

also need to mention this in vim_diff.txt

Reply:
A bit misleading, they were always "drawn" left to right, what changes here is the sort order. Also which signs are visible/not visible does not change here. How about:

```suggestion
• The order in which signs are placed was changed. Higher priority signs will now appear left of lower priority signs.
```

What we already have in vim_diff.txt also does not seem like it needs to change; there is no order in Vim:

      'signcolumn'  supports up to 9 dynamic/fixed columns

Mentionioning the intention of the sort order somewhere `sign.txt` or at `:h signcolumn` in seems more relevant.